### ISSUE TITLE
[6.0.5] imagesetpixel() expects parameter 1 to be resource, boolean given.... in Log file

### ISSUE THREAD

**Issue by cdamken** [owncloud/core#12171]
### Expected behavior
No error

### Actual behavior
I have 126'111'365 times this error in the log file:
```
{"app":"PHP","message":"imagesetpixel() expects parameter 1 to be resource, boolean given at \/var\/www\/html\/owncloud\/lib\/private\/image.php#694","level":3,"time":"2014-11-11T13:48:02+00:00"}
```
This error was realized by me yesterday and it is written the whole time in the log file (like ~100 times per second).

### Server information
RHEL 6
oC6.0.5
php 5.3.3
Open LDAP
MySQL 5.1.73

00002078

**Comment by MorrisJobke** [comment]
@cdamken the fix will be in 6.0.7

**Comment by cdamken** [comment]
@MorrisJobke  Thank you!, I will test it when the 6.0.7 is out :+1:
